### DISCUSSION TITLE
How do I populate Benchmarks ("Compare with....")?

### DISCUSSION THREAD

**Discussion by sawasy** [ghostfolio/ghostfolio#2367] [Q&A] [answered]
I am using docker-compose to run an instance. I've figured out most things, but I am unable to figure out how to populate the "Compare with..." drop down under the Analysis section. Any help appreciated.

Cheers

m!

**Comment by dtslvr** [comment, marked as answer]
Hi @sawasy!

This feature is called _Benchmarks_.

Please go to _Admin Control_ → _Market Data_ and select an asset profile. In the dialog, there is a _Benchmark_ checkbox.

**Comment by sawasy** [comment]
So close! Fantastic. Thank you. I had managed to add, say, S&P 500 and gather the historical data, but didn't notice the Set Benchmark option. 

m!

**Comment by ups7** [comment]
This is very useful, thank you @dtslvr! I had been wondering this myself but never had the chance to ask.

I think there are a lot of hidden features in Ghostfolio which are sometimes a bit hard to find or figure out without any documentation. These should ideally be in an FAQ on the website, and will allow people to make the most of Ghostfolio.

**Comment by prashmohan** [comment]
Thank you @dtslvr . I was looking for this as well.

Is it possible to backfill data for a benchmark? I setup ^SP500TR as the benchmark for SP500 total returns. But it seems to only populate the data from when I added it into market data.

**Reply by dtslvr** [reply]
@prashmohan, have you tried to run _Gather Historical Data_ from the same menu as _Set as Benchmark_?

**Reply by prashmohan** [reply]
Sweet. That does it exactly.
